.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+* @softinio
```

Comment:
I would assign zio-schema team here. We can create one if it doesn't already exist.

Reply:
I've update this to `@zio/zio-schema`.